.github/workflows/push-build.yaml

```diff
@@ -75,3 +97,7 @@ jobs:
           GITHUB_REF: $ {{ env.GITHUB_REF }}
         run: |
           tests/scripts/build-release.sh
+
+      - name: push to other OCI
+        run: |
+          tests/scripts/test-ci.sh push
```

Comment:
I know fully integrating with the makefile is more challenging, but can we at least call this script from the makefile? Then anytime `make publish` is called, we know all the resources are properly pushed. Our official builds are anyway only triggered by this workflow so it's not a big difference, but it still feels like it should be called from the makefile instead of from this workflow.

Reply:
done